src/extension-test/unit/debugger/util-test.ts

```diff
@@ -17,63 +16,119 @@ function getCoordinates(text: string): (string | number)[] {
-      const docText = getTestFileText(fileName);
-      debugResponse.coor = getCoordinates(docText);
-      doc.insertString(docText);
+    function expectBreakpointToBeFound(coorsAndBody: string) {
+      debugResponse.coor = getCoordinates(coorsAndBody);
+      const doc = new model.StringDocument();
+      doc.insertString(coorsAndBody);
       const tokenCursor = doc.getTokenCursor(0);
       moveTokenCursorToBreakpoint(tokenCursor, debugResponse);
       expect(tokenCursor.getPrevToken().raw.endsWith('|')).toBe(true);
     }
 
     it('simple example', () => {
-      expectBreakpointToBeFound('simple.clj');
+      expectBreakpointToBeFound(`;; 3, 2
+      (defn simple [x]
+        (+ 1 #break x|))`);
     });
 
     it('function shorthand', () => {
-      expectBreakpointToBeFound('fn-shorthand.clj');
+      expectBreakpointToBeFound(`;; 3, 1, 2, 2
+      (defn fn-shorthand
+        [s x]
+        (filter #(= % #break x|) s))`);
     });
 
-    it('map', () => {
-      expectBreakpointToBeFound('map.clj');
-    });
+    describe('maps', () => {
+      it('map', () => {
+        expectBreakpointToBeFound(`;; 3, "[1 (+ 1 2)]", 2
+        (defn test-map
+          [x]
+          {:hello "world" [1 (+ 1 2)] (+ 5 #break x|)})`);
+      });
 
-    it('metadata symbol', () => {
-      expectBreakpointToBeFound('metadata-symbol.clj');
-    });
+      it('metadata map', () => {
+        expectBreakpointToBeFound(`;; 3, 2, 2, 2
+        (defn test-metadata-symbol
+          [x]
+          (let [y x]
+            ^{:hello "world"}
+            (+ x ^{:inner "meta"} (+ 1 #break y|))))`);
+      });
 
-    it('metadata map', () => {
-      expectBreakpointToBeFound('metadata-map.clj');
+      it('metadata map last sexp', () => {
+        expectBreakpointToBeFound(`;; 3, 2, 2, 2
+        #dbg
+         (defn test-metadata-symbol
+           [x]
+           (let [y x]
+             ^{:hello "world"}
+             (+ x ^{:inner "meta"} (+ 1 y|))))`);
+      });
     });
 
-    it('metadata map last sexp', () => {
-      expectBreakpointToBeFound('metadata-map-last.clj');
+    it('metadata symbol', () => {
+      expectBreakpointToBeFound(`;; 3, 2, 2
+      (defn test-metadata-symbol
+        [x]
+        (let [y x]
+          (+ ^String x #break ^String y|)))`);
     });
 
     it('ignored forms', () => {
-      expectBreakpointToBeFound('ignored-forms.clj');
+      expectBreakpointToBeFound(`;; 3, 1, 1
+      (defn test-ignore
+        [x]
+        (let [y #_#_#_2 3 4 #break x|]
+          (+ x y)))`);
     });
 
     it('syntax quote', () => {
-      expectBreakpointToBeFound('syntax-quote.clj');
+      expectBreakpointToBeFound(`;; 3, 2, 1, 2, 1, 2
+      (defn test-syntax-quote
+        [y]
+        \`{:hello ~(+ 1 #break y|)})`);
+    });
+
+    describe('derefs', () => {
+      it('atom', () => {
+        expectBreakpointToBeFound(`;; 3, 1
+        #dbg
+        (defn boom
+          []
+          @!a|)`);
+      });
```

Comment:
@jpmonettas , I now realize you may be referring to this test. It passes just fine.